### PR TITLE
Overview -> Dev

### PR DESCRIPTION
**Pull request by FangzhuoXi**
created a basic view of image gallery
fixed a bug from add to cart section
add some css styles

### CODE REVIEW

File: src/components/Overview/ImageGallery.jsx [hunk 1]
```diff
@@ -1,7 +1,70 @@
 import React, {useEffect, useState} from 'react';
+import styled from 'styled-components';
+import { FaAngleLeft, FaAngleRight, FaAngleUp, FaAngleDown } from 'react-icons/fa';
 
-const ImageGallery = () => {
-  return (<div className='ImageGallery'>ImageGallery</div>)
+const Img = styled.img`
```
Comment (Parkerjn90) [inline src/components/Overview/ImageGallery.jsx:5]: I would recommend putting all your styles in one file and then importing them into the files you need - just makes for a cleaner look

File: src/components/Overview/Overview.jsx [hunk 2]
```diff
@@ -11,8 +11,8 @@ import OverviewStyles from './../../../styles/OverviewStyle.css';
 const Overview = ({productId, productDetails}) => {
   const [styles, setStyles] = useState([])
   const [currentStyle, setCurrentStyle] = useState(styles[0])
-  // console.log('styles: ', styles);
-  // console.log('currentStyle: ', currentStyle);
+
+  console.log(currentStyle);
```
Comment (Parkerjn90) [inline src/components/Overview/Overview.jsx:15]: Don't forget to take your console logs out before a pull request

File: src/components/Overview/ImageGallery.jsx (hunk 1 above)
Comment (FangzhuoXi) [reply src/components/Overview/ImageGallery.jsx:5]: So can you import a styled component? I'm using 2 different method to style. Not sure if I'm able to put everything in one file

File: src/components/Overview/AddToCart.jsx [hunk 3]
```diff
@@ -9,8 +9,8 @@ const AddToCart = ({currentStyle}) => {
   console.log('currentStyle', currentStyle);
   console.log('sizeSelected', sizeSelected);
```
Comment (Parkerjn90) [inline src/components/Overview/AddToCart.jsx:10]: don't forget to take out console logs before pull requests

**Pull request merged**